### PR TITLE
fix: Fix protobuf import error

### PR DESCRIPTION
**Pull request by noisrucer**
- pb2 files import from `impaction_ai/v1alpha/...`, causing an error.
   - Alternative: Since pb2 files are not meant to be edited manually, put `ingestion/v1alpha` in the root directory and include it in `pyproject.toml`.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -20,6 +21,7 @@ black = "^23.7.0"
 mypy = "^1.5.1"
 isort = "^5.12.0"
 pre-commit = "^3.5.0"
+python-dotenv = "^1.0.0"
```
Comment (kde713) [inline pyproject.toml]: We don't use `python-dotenv` in SDK. Please rollback this change.

**Review by kde713 — CHANGES_REQUESTED** [review]

File: alignai/config.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ def __init__(
         max_buffer_size: int = MAX_BUFFER_SIZE,
         api_max_retries: int = API_MAX_RETIRES,
     ):
-        """impaction.ai client configuration to adjust the behavior of the SDK.
+        """alingai client configuration to adjust the behavior of the SDK.
```
Comment (kde713) [inline alignai/config.py]:
```suggestion
        """Align AI client configuration to adjust the behavior of the SDK.
```

File: pyproject.toml (hunk 1 above)
Comment (noisrucer) [reply pyproject.toml]: Commit: 9ba248dd317d933d4065c1980342c1d5c66b62c2

File: alignai/config.py (hunk 2 above)
Comment (noisrucer) [reply alignai/config.py]: Commit: 9ba248dd317d933d4065c1980342c1d5c66b62c2

**Review by kde713 — APPROVED** [review]

**Pull request merged**